Tests/MetaTests/Visitors/ImportVisitorTests.swift

```diff
@@ -0,0 +1,18 @@
+import XCTest
+import Meta
+import SwiftSyntax
+
+class ImportVisitorTests: XCTestCase {
+    func testFetchesAllImports() throws {
+        let input = URL(fileURLWithPath: String(#file))
+        let visitor = ImportVisitor()
+        
+        let ast = try SyntaxTreeParser.parse(input)
+        ast.walk(visitor)
+        
+        let imports = visitor.imports.map { $0.description.trimmingCharacters(in: .whitespacesAndNewlines) }.sorted()
```

Comment:
just as idea, I suppose you usually use `imports` as String instead to `ImportDeclSyntax` - maybe it will be interesting to move this `trimming` code to `ImportVisitor` to prevent duplicate and differents ways to clean up

Reply:
Changed to `Set<String>`